src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+$color-active-title:#34568b;
+$color-border-card: #f3f3f3;
```

Comment:
Now the variables are named correctly, but it will be even better if the type of the variable is at the end of the name (`$main-accent-color`, `$secondary-color`, etc.).